criu/include/restorer.h

```diff
@@ -50,6 +50,11 @@ struct restore_posix_timer {
 	int overrun;
 };
 
+struct restore_child_subreaper {
+	bool has_child_subreaper;
```

Comment:
why do you need this has_child_subreaper? child_subreaper is PID, so it is always greater than 0. If child_subreaper is zero, child_subreaper isn't set. Hm?

Reply:
`child_subreaper` is not PID.

Reply:
Ok. It is even better:). child_subreaper can be 0 or 1. If it is 0 we need to do nothing. If it is one we need to call prctl(PR_SET_CHILD_SUBREAPER). I still don't understand why we need has_child_subreaper.